raiden/tests/fixtures/variables.py

```diff
@@ -333,7 +331,7 @@ def private_rooms():
 @pytest.fixture
 def environment_type():
     """Specifies the environment type"""
-    return Environment.PRODUCTION
+    return Environment.DEVELOPMENT
```

Comment:
Use latest version of contracts in tests. This is needed since the production mode still uses an old version of the contract where the interfaces in the new contract version were broken.

Reply:
Then this is a breaking change that is not backwards compatible. When will we release this? If we do, I think it's the right time to also do https://github.com/raiden-network/raiden/issues/3254

Reply:
hmm... i think it is backwards compatible because for people running Red eyes, using newer version of Raiden with this change is not going to break their existing nodes as they use "production" mode and that is defined in Raiden as `0.4.0`

Reply:
We need a definition for backwards incompatibility. I remember we did discuss this in person, but I'm lacking the vocabulary to talk about this.

My understand persists though, this will not work with previous clients so we might as well fix the messages.

Reply:
As discussed in the office, this is only for the Development contracts. Which means not the ones deployed on the mainnet. The point of breaking compatibility with red eyes is when we switch the `Environment.PRODUCTION` variable to point to anything other than Red Eyes.

So that would be the best point to have the message changes and any other backwards compatibility breaking changes we can fit in.